mpact/services.py

```diff
@@ -262,6 +262,15 @@ def schedule_messages(xlsx_file):
 
     def is_blank(val):
         return val is None or val == ''
+    
+    def is_positive_numeric(val):
+        try:
+            val=int(val)
+            if val >= 0:
+                return True
+        except ValueError:
+            pass
+        return False
```

Comment:
nit: could also put this in the except clause

Reply:
Good point. I think that wasn't true before @kaapstorm 's suggestion of checking that it is a positive integer in a single return statement.